MAUI/DataForm/Data-annotations.md

```diff
@@ -0,0 +1,475 @@
+---
+layout: post
+title: Data annotations in .NET MAUI DataForm(SfDataForm) control | Syncfusion
+description: Learn here all about Data annotations support in Syncfusion Maui DataForm (SfDataForm) control to handle the data.
+platform: maui
+control: SfDataForm
+documentation: UG
+---
+
+# Data annotations in .NET MAUI DataForm (SfDataForm)
+
+The data form supports the following attribute to handle the data, and these attributes can be accessible using `System.ComponentModel.DataAnnotation` assembly.
+
+## Display attribute
+
+<table>
+<tr>
+<th>Property</th>
+<th>Details</th>
+</tr>
+<tr>
+<td>
+{{'[Name] `Link`
+</td>
+<td>
+Specifies the label text.
+</td>
+</tr>
+<tr>
+<td>
+{{'[GroupName]`Link`
+</td>
+<td>
+Specifies the group name which groups the fields in the data form. Refer to  `Link` for more details.
+</td>
+</tr>
+<tr>
+<td>
+{{'[ShortName] `Link`
+</td>
+<td>
+Specifies the label text. It takes higher priority than Name.
+</td>
+</tr>
+<tr>
+<td>
+{{'[AutoGenerateField]`Link`
+</td>
+<td>
+Specifies whether the field should be auto generated or not. Refer to `Link` for more details.
+</td>
+</tr>
+<tr>
+<td>
+{{'[ResourceType]`Link`
+</td>
+<td>
+Specifies the Resources File (.Resx) which is used to localize the Display attribute of `Name`, `ShortName`, `GroupName` and `Prompt` values.
```

Comment:
Specifies the Resources File (.Resx) which is used to localize the Display attribute of `Name`, `ShortName`, `GroupName`, `Prompt` and `ItemsSource` values.

Reply:
ItemsSource added